far/vmenu.hpp

```diff
@@ -53,25 +53,25 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //----------------------------------------------------------------------------
 
 // Цветовые атрибуты - индексы в массиве цветов
-enum vmenu_colors
+enum class vmenu_colors
```

Comment:
Do we actually need this in .hpp?

Reply:
Down below, we have

```C++
FarColor Colors[std::to_underlying(vmenu_colors::COUNT)]{};
```

Is there a way to get the number of Far colors (15) without exposing this definition?

Reply:
vector maybe?

Reply:
Did it another way. Please see if you like it.

Reply:
It's ok, as long as we assert the equality.

Reply:
We do. Resolving this one.